documents/policies/external/Competition Requirements.md

```diff
@@ -119,6 +138,11 @@ The WCA Delegate must submit the following information when requesting approval
    2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
       1. Other events may only be held unofficially.
    3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information.
+      1. The main event should be the main focus of the competition by being recognizable in the competition schedule and having an appropriate proportion of rounds and attempts.
+   4. An additional event may be added upon the WCA Delegate's request at the discretion of the WCAT. The competition must meet the following requirements for an additional event to be added:
+      1. The competition is at least 7 days away.
+      2. A registration period for the competition must have a minimum of 72 hours before it closes.
```

Comment:
If the competition is e.g. 10 days away but registration is open for only another 48 hours would it be possible to extend it? It would be good to note this, e.g. "The registration period may be extended if it is still open if this is not the case" or "It is not possible to extend registration in order to enable another event to be added".

Reply:
I tried to clarify this earlier in the WCRP that registration periods may be extended. Like Clement mentioned, the WCAT would not deny extending an active registration period so this may be done in order to fulfill the requirements of adding an event.